### ISSUE TITLE
Smooth Transaction Experience with Room for Feature Enhancements

### ISSUE THREAD

**Issue by hyderqadir** [beyond-btc/feedback#19]
### Contact details

dgskillsvaam@gmail.com

### Describe the solution you'd like

I tested the bridge platform and overall the experience was smooth with quick transaction activity. The interface is easy to use.

That said I noticed a few areas where improvements could be made. First adding an "estimated fees" feature would be very useful helping users see the transaction costs before confirming the trade. Another suggestion is to display the balance upfront when selecting tokens, along with a "Max" button to allow users to easily swap their full balance without manual input. These additions would enhance the overall usability and efficiency.

In summary, the platform works well but with these added features, it could provide an even better user experience.

**Comment by sunnybeyond** [comment]
Estimated fees is useful indeed. Max botton too, but then it should keep enough funds for gas fees.

**Comment by sunnybeyond** [comment]
**Congratulations!** Your review has passed our quality check, and you are now eligible for a The Orange Pills whitelist! To register your address, please follow these steps:

1. Join the Beyond Discord server. (https://join.beyond.tech/) 
2. Open a support ticket in the #support channel.
3. Add a comment to your GitHub entry with the ticket number from your Discord support ticket.
4. In the support ticket, make sure to include:
- Your Taproot address (bc1p)
- A link to your GitHub entry

**Important:**

- Entries missing any of the required information above will NOT be considered and will be closed automatically.
- All entries must be completed by Sunday, November 3rd, 23:59 UTC. Late entries will NOT be accepted.

**Comment by hyderqadir** [comment]
Thank You.. 

Ticket # 836
